app/controllers/api/v1/sessions_controller.rb

```diff
@@ -41,20 +41,27 @@ def session_params
       end
 
       def sign_in(user)
+        user.generate_session_token!(extended_session: session_params[:extend_session])
+
         # Creates an extended_session cookie if extend_session is selected in sign in form.
         if session_params[:extend_session]
           cookies.encrypted[:_extended_session] = {
```

Comment:
I suppose that we are intending to eventually remove the [:_extended_session] cookie since we can manually set up the expiry in User `generate_session_token!(extended_session: false)`?

Reply:
Nope - we need the cookie to store the value on the client side, or else there's no way for us to tell which user is signed in